tools/acorn-optimizer.js

```diff
@@ -1444,9 +1606,11 @@ var registry = {
   applyDCEGraphRemovals: applyDCEGraphRemovals,
   minifyWhitespace: function() { minifyWhitespace = true },
   noPrint: function() { noPrint = true },
+  last: function() {},
```

Comment:
What is this?

Reply:
The python code uses "last" as a marker for knowing that nothing will run next. The acorn code doesn't actually need it, as you can see here, but adding this lets the existing python driver code work unmodified. Basically this makes the new pass implement the exact same API surface as the old pass.

Reply:
(A refactoring on the python side could remove it, i'll add a TODO)